ui/components/multichain/connected-accounts-menu/connected-accounts-menu.tsx

```diff
@@ -67,13 +95,30 @@ export const ConnectedAccountsMenu = ({
           <TsMenuItem
             iconName={IconName.SwapHorizontal}
             data-testid="switch-account-menu-item"
+            onClick={() => {
+              trackEvent({
+                category: MetaMetricsEventCategory.Navigation,
+                event: MetaMetricsEventName.NavAccountSwitched,
+                properties: {
+                  location: 'Main Menu',
```

Comment:
So, metametrics are specific to component. I don't think we should use these here. We might create new one. Not a part of this ticket